### DISCUSSION TITLE
Implement ILR/CRISP methods for estimating response propensity

### DISCUSSION THREAD

**Discussion by vladber-bls** [ncn-foreigners/nonprobsvy#54] [Ideas]
The attached presentation proposes simple and efficient method for estimating response propensity to NPS. It may be more efficient than Chen, Li, and Wu (2020) method in many practically relevant cases. In case of large overlap between PS and NPS, it 
allows straightforward improvement of the estimates  if the overlapping units can be linked.    

[A2_Tue_1100_Beresovsky.pdf](https://github.com/user-attachments/files/15915712/A2_Tue_1100_Beresovsky.pdf)

**Comment by BERENZ** [comment]
Thanks for the proposal! To the end of August we will be working on making the package more robust to errors and computationally efficient so then we can focus on implementing new methods.

**Reply by vladber-bls** [reply]
Dear nonprobsvy Team. We recently resubmitted revision of our paper to one of the refereed journals. Meanwhile, we also published the paper in Arxiv:  

http://arxiv.org/abs/2312.05383

We reviews and compared the proposed quasi-randomization methods, addressing the matters of efficiency, consistency, correlations of the observed indicators, overlap, etc.

**Reply by BERENZ** [reply]
@vladber-bls I just uploaded the updated version of the package and finally I can focus on implementing your approach!
